src/diagnostic/diagnostic_manager.h

```diff
@@ -67,88 +78,77 @@ class DiagnosticsManager : public IDiagnosticsManager
         return dMan;
     }
 
-    void dump(double timeStamp, double timeStep) override;
-    void dump_level(std::size_t level, double timeStamp) override;
 
-    DiagnosticsManager& addDiagDict(PHARE::initializer::PHAREDict const& dict);
+    DiagnosticsManager& addDiagDict(initializer::PHAREDict const& dict);
 
 
-    DiagnosticsManager& addDiagDict(PHARE::initializer::PHAREDict&& dict)
-    {
-        return addDiagDict(dict);
-    }
+    DiagnosticsManager& addDiagDict(initializer::PHAREDict&& dict) { return addDiagDict(dict); }
+
 
+    auto& diagnostics() const { return diagnostics_; }
+
+
+    Writer& writer() { return *writer_.get(); }
+
+
+    DiagnosticsManager(const DiagnosticsManager&)  = delete;
+    DiagnosticsManager(const DiagnosticsManager&&) = delete;
```

Comment:
redundant const?

Reply:
I just moved these lines, didn't add anything. you suggest removing const in the && version? (I'd agree)

Reply:
:man_shrugging:

Reply:
https://en.cppreference.com/w/cpp/language/move_constructor